package.json

```diff
@@ -13,9 +13,7 @@
     },
     "scripts": {
         "lint": "eslint .",
-        "test": "istanbul cover _mocha -- test/* -R spec",
-        "test:config": "mocha test/config",
-        "test:example": "mocha test/example"
+        "test": "./node_modules/mocha/bin/mocha ./test/*.js"
```

Comment:
Yes let me add a new one for specifically coverage. I don't think it's good for us to use `npm run test` with coverage on since `istanbul` is not the most stable libs out there.